py4cast/models/vision/unetrpp.py

```diff
@@ -304,8 +304,7 @@ def __init__(
         in_channels=4,
         dropout=0.0,
         transformer_dropout_rate=0.1,
-        subsampling_rate: int = 4,
-        **kwargs,
+        downsampling_rate: int = 4,
```

Comment:
int=4 as default seems quite aggressive (mere remark)

Reply:
It is the default value of the reference implementation. I tend to stick to those as defaults. Maybe if we identify a good candidate we will switch to its settings ?

Reply:
let's keep it this way, my remark was no realy to the point